src/Wallet/Cip30.purs

```diff
@@ -82,12 +81,15 @@ getWalletAddress :: Cip30Connection -> Aff (Maybe Address)
 getWalletAddress conn = fromHexString _getAddress conn <#>
   (_ >>= addressFromBytes <<< rawBytesAsCborBytes)
 
-getCollateral :: Cip30Connection -> Aff (Maybe TransactionUnspentOutput)
-getCollateral conn = fromMaybeHexString getCip30Collateral conn >>=
-  case _ of
-    Nothing -> pure Nothing
-    Just bytes -> do
-      liftEffect $
+getCollateral :: Cip30Connection -> Aff (Maybe (Array TransactionUnspentOutput))
```

Comment:
A commentary could be useful here. Also when does it return `Nothing` and when does it throw